code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -95,7 +95,7 @@
 
 /datum/species/human/felinid/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/M)
 	.=..()
-	if(chem.type == /datum/reagent/consumable/cocoa)
+	if(istype(chem, /datum/reagent/consumable/cocoa) || istype(chem, /datum/reagent/consumable/cocoa/hot_cocoa))
```

Comment:
If you have
A/B/C
and 
A/B/C/D
istype(item, A/B/C) will return true for types A/B/C and A/B/C/D
istype(item, A/B/C/D) will return true for type A/B/C/D